### PR TITLE
Discuss masonry

### PR DESCRIPTION
**Pull request by mattkrick**
burndown
- [x] reflection grid
- [x] task grid

TEST
- [ ] do a retro where every card has a different # of lines, put them all in the same group & discuss the group. the reflections are in a masonry
- [ ] create 2+ rows of tasks, the tasks are in a masonry
- [ ] resize the window, it works
- [ ] add an extra couple lines to a task in the first row, the grid updates correctly
- [ ] run an action meeting & create 2+ rows of tasks, it works

### CODE REVIEW

File: src/universal/components/TaskEditor/TaskEditor.js [hunk 1]
```diff
@@ -168,41 +190,11 @@ class TaskEditor extends Component {
           ref={setEditorRef}
         />
         {renderModal && renderModal()}
-      </div>
+      </RootEditor>
     )
   }
 }
 
-const styleThunk = () => ({
```
Comment (mattkrick) [inline src/universal/components/TaskEditor/TaskEditor.js]: the async styles were causing the grid measurements to fail so i had to remove aphrodite from all of these

File: package.json [hunk 2]
```diff
@@ -222,6 +222,7 @@
     "redux-form": "^7.3.0",
     "redux-raven-middleware": "^1.2.0",
     "redux-thunk": "2.2.0",
+    "resize-observer-polyfill": "^1.5.0",
```
Comment (mattkrick) [inline package.json]: only chrome supports this so far

**Comment by mattkrick** [comment]
@jordanh ready for your review!

**Comment by jordanh** [comment]
## Test Cycle 2
- [x] do a retro where every card has a different # of lines, put them all in the same group & discuss the group. the reflections are in a masonry
- [ ] create 2+ rows of tasks, the tasks are in a masonry
- [ ] resize the window, it works
- [ ] add an extra couple lines to a task in the first row, the grid updates correctly
- [ ] run an action meeting & create 2+ rows of tasks, it works

**Comment by jordanh** [comment]
## Test Cycle 2
- [x] do a retro where every card has a different # of lines, put them all in the same group & discuss the group. the reflections are in a masonry
- [x] create 2+ rows of tasks, the tasks are in a masonry
- [x] resize the window, it works
- [x] add an extra couple lines to a task in the first row, the grid updates correctly
- [x] 🗒 run an action meeting & create 2+ rows of tasks, it works

### Test notes 🗒 

While processing an agenda item during an action meeting, the tasks didn't collapse in the same way as during a retro. I'm guessing because the container hasn't been designed that way. S'okay. Just wanted to note it.

**Pull request merged**